network/edge-services/how-to/subscribe-edge-services.mdx

```diff
@@ -0,0 +1,71 @@
+content:
+  h1: How to subscribe to Edge Services
+  paragraph: Find out how to take your first steps with Scaleway Edge Services by subscribing to a pricing plan. Learn how to choose the best plan for your needs and change your plan at the click of a button.
+dates:
+  validation: 2024-10-15
+  posted: 2024-10-15
+tags: object-storage edge-services subscription-plan subscribe billing pricing
+categories:
+  - network
+---
+
+Edge Services is an additional feature for Scaleway Load Balancers and Object Storage buckets. It lets you  benefit from a powerful caching service to reduce the load on your origin servers and streamline delivery. On top of this, you get a customizable Edge Services endpoint and can set it to a subdomain of your choice, securing it with an SSL/TLS certificate.
+
+To use Edge Services, you must subscribe to a [pricing plan](https://www.scaleway.com/en/pricing/network/#edge-services). Within its monthly price, each pricing plan includes:
+
+- A fixed number of Edge Services [pipelines](/network/edge-services/concepts/#pipeline). You can create pipelines for either Load Balancers, Object Storage buckets, or a mixture of both, with your subscription plan.
+- A certain amount of egress [cache](/network/edge-services/concepts/#cache) data (the quantity of data transferred from Edge Services' caches, not including the transfer from the origin bucket or Load Balancer).
+- A custom domain and SSL certificate (managed or custom) for each pipeline.
+
+If you create more pipelines than are included in your plan, or your pipelines' caches egress more data than is included, you will be charged additionally for this. The rates per pipeline/GB of data are indicated on the [pricing](https://www.scaleway.com/en/pricing/network/#edge-services) page.
+
+Subscriptions are pro-rata, meaning you can cancel your subscription or change your plan at any time, and you will be charged proportionally.
+
+Find out more about how Edge Service subscription plans and billing works on our [Understanding Edge Services pricing](/network/edge-services/reference-content/understanding-edge-pricing/) page.
+
+## How to subscribe to Edge Services
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+
+<Message type="tip">
+You can subscribe to Edge Services from the Object Storage or Load Balancer dashboards of the Scaleway console. However, the instructions below detail how to describe from the dedicated Edge Services section of the console. Whichever path you use, the same pricing plans and subscription mechanisms apply. Subscribing from the Object Storage or Load Balancer dashboards does **not** limit you to only creating pipelines for that product with your subscription.
```

Comment:
Maybe we should say somewhere that Edge Services is billed per project. Thus, a user can enable ES on 1 orga, and disable it on another, or have 2 different plans for 2 different orgas